src/components/Page/components/Header/components/Title/Title.tsx

```diff
@@ -6,7 +6,6 @@ import type {AvatarProps} from '../../../../../Avatar';
 import type {ThumbnailProps} from '../../../../../Thumbnail';
 import {classNames} from '../../../../../../utilities/css';
 import {useFeatures} from '../../../../../../utilities/features';
-import {useMediaQuery} from '../../../../../../utilities/media-query';
```

Comment:
This hook allows react to change when the mobile menu icon shows. In this case we do not want to change the font size on this breakpoint.